src/main/java/com/softserveinc/ita/utils/runners/ElementsUtil.java

```diff
@@ -1,50 +1,22 @@
 package com.softserveinc.ita.utils.runners;
 
 import com.codeborne.selenide.CollectionCondition;
-import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
 import lombok.experimental.UtilityClass;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.Select;
 import java.util.List;
 
-import java.util.Random;
-
-import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
-import static com.codeborne.selenide.Configuration.*;
-import static com.codeborne.selenide.Selenide.*;
 import static java.time.Duration.ofSeconds;
 import static java.util.stream.Collectors.toList;
 
 @UtilityClass
 public class ElementsUtil {
 
-    public void selectPriceFilterFromModalMenu(int value, String selector) {
-        var selectCheap = $(selector).should(Condition.appear);
-        var index = new Select(selectCheap);
-        index.selectByIndex(value);
-    }
-
-    public void setPriceValueInFilter(String selector, String price) {
-        var maxPriceField = $x(selector);
-        timeout = 5000;
-        maxPriceField.click();
-        maxPriceField.setValue(price);
-    }
-
-    public void randomizerForListCategories(String selector) {
-        var random = new Random();
-        var list = $$x(selector)
-                .shouldBe(sizeGreaterThan(0), ofSeconds(10));
-        list
-                .get(random.nextInt(list.size()))
-                .click();
-    }
-
-    public List<String> getListWithGoods(String selector, String item) {
-        return $$x(selector)
+    public static List<String> getListWithGoods(ElementsCollection selector, String item) {
```

Comment:
these operations are related only to SearchResultPage, and it would not be used anywhere else, in ElementsUtil should be located common things (which could be used from/for any place) like

ex: public static void setInput(SelenideElement element, String value){
element.click();
element.clear();
element.setValue(value);
}